### ISSUE TITLE
Missing aws-native.apigatewayv2.Stage class

### ISSUE THREAD

**Issue by jasonlim-bandlab** [pulumi/pulumi-aws-native#1692]
## Hello!

- Vote on this issue by adding a 👍 reaction
- If you want to implement this feature, comment to let us know (we'll work with you on design, scheduling, etc.)

## Issue details

Why is there no `aws-native.apigatewayv2.Stage` class? We have it in AWS Classic, so am I missing something in the documentation? It doesn't make sense that the Stage class is unavailable, as we should be able to create different stages for our API Gateway.

**Comment by t0yv0** [comment]
It appears that Cloud Control API does not yet support this resource per https://docs.aws.amazon.com/cloudcontrolapi/latest/userguide/supported-resources.html

**Comment by jasonakon** [comment]
@t0yv0, if it's not supported by Cloud Control API at the moment, are you suggesting we temporarily use `@pulumi/aws` with `aws.apigatewayv2.Stage`? Or do you have a better method for configuring it using `@pulumi/aws-native`? Please let me know.

**Comment by t0yv0** [comment]
Indeed, it's not going to be supported in `@pulumi/aws-native` until such time as Cloud Control API supports it. If you need to manage these resources with Pulumi, `@pulumi/aws` is indeed the recommendation.

**Comment by pgavlin** [comment]
All upstream issues have been fixed; closing.

**Comment by pgavlin** [comment]
This was closed in error. We're still waiting on support for this resource in Cloud Control.
